### ISSUE TITLE
lower_text position on view mode Month is wrong on big timeline

### ISSUE THREAD

**Issue by nhandq-dev** [frappe/gantt#212]
Because the dates of the month, not always exactly 30 so when we have a long timeline, the x position of Month_lower could be wrong if you calculating this way

```
const base_pos = {
  x: i * this.options.column_width,
  lower_y: this.options.header_height,
  upper_y: this.options.header_height - 25,
}
```

Because in Month mode `this.options.column_width` always be 30 so we have a timeline from begin of 2020 to the end of 2024 the last base_pos.x could be 120*60=7200 but the exact number must be 7304, and so we have this chart.
![image](https://user-images.githubusercontent.com/56851883/106583480-ba893580-6577-11eb-80fe-654d19b93e9c.png)

You guy can consider changing it to this way

```
const base_pos = {
  lower_y: this.options.header_height,
  upper_y: this.options.header_height - 25,
}

if (this.view_is(VIEW_MODE.MONTH)) {
  base_pos.x = date_utils.diff(date, this.gantt_start, 'day') * this.options.column_width / 30
} else {
  base_pos.x = i * this.options.column_width
}

const x_pos = {
  ...,
  Month_lower: (date_utils.get_days_in_month(date) * this.options.column_width / 30) / 2,
  ...,
}
```

**Comment by hiawui** [comment]
I've made a commit to fix this bug #270

**Comment by MariaBee3** [comment]
Thanks, I've used this fix and it worked perfectly!
